error running FilterBunches

```
hpsmc.job:INFO Executing 'filter_bunches' with command: java -Xmx1g -XX:+UseSerialGC -Dorg.lcsim.cacheDir=/export/scratch/tmp.CFlQcDiXKn/scratch -cp /usr/local/hps/java/distribution/target/hps-distribution-5.2-SNAPSHOT-bin.jar org.hps.util.FilterMCBunches -e 250 idm_rot.slcio idm_rot_filt.slcio -a
================ Component: filter_bunches ================
Error: Could not find or load main class org.hps.util.FilterMCBunches
Caused by: java.lang.ClassNotFoundException: org.hps.util.FilterMCBunches
hpsmc.job:INFO Execution of filter_bunches took 0.3157 second(s) with return code: 1
Traceback (most recent call last):
  File "/usr/local/lib/python/hpsmc/job.py", line 832, in <module>
    job.run()
  File "/usr/local/lib/python/hpsmc/job.py", line 516, in run
    self._execute()
  File "/usr/local/lib/python/hpsmc/job.py", line 574, in _execute
    raise Exception("Output file '%s' is missing after execution." % outputfile)
Exception: Output file 'idm_rot_filt.slcio' is missing after execution.
```
Just saving this so I don't forget about it.

--- a/context/Dockerfile
+++ b/context/Dockerfile
@@ -13,7 +13,8 @@ RUN mkdir ${__prefix}/hps &&\
       -Dcheckstyle.skip \
     &&\
     cp -t ${__prefix}/hps/java/ \
-      ~/.m2/repository/org/lcsim/lcio/2.7.4-SNAPSHOT/lcio-2.7.4-SNAPSHOT-bin.jar
+      ~/.m2/repository/org/lcsim/lcio/2.7.4-SNAPSHOT/lcio-2.7.4-SNAPSHOT-bin.jar &&\
+    find distribution/target -type f -name '*-bin.jar'
 ENV HPS_JAVA_BIN_JAR=${__prefix}/hps/java/distribution/target/hps-distribution-5.2.0-bin.jar
 
 # download and unpack fieldmaps

--- a/context/Dockerfile
+++ b/context/Dockerfile
@@ -14,7 +14,7 @@ RUN mkdir ${__prefix}/hps &&\
     &&\
     cp -t ${__prefix}/hps/java/ \
       ~/.m2/repository/org/lcsim/lcio/2.7.4-SNAPSHOT/lcio-2.7.4-SNAPSHOT-bin.jar
-ENV HPS_JAVA_BIN_JAR=${__prefix}/hps/java/distribution/target/hps-distribution-5.2-SNAPSHOT-bin.jar
+ENV HPS_JAVA_BIN_JAR=${__prefix}/hps/java/distribution/target/hps-distribution-5.2.0-bin.jar
 
 # download and unpack fieldmaps
 #   destination is chosen to be what is expected from hps-mc defaults

--- a/context/container.cfg
+++ b/context/container.cfg
@@ -3,7 +3,7 @@
 
 [DEFAULT]
 hpsmc_dir = /usr/local/
-hps_java_bin_jar = /usr/local/hps/java/distribution/target/hps-distribution-5.2-SNAPSHOT-bin.jar
+hps_java_bin_jar = /usr/local/hps/java/distribution/target/hps-distribution-5.2.0-bin.jar
 lcio_bin_jar = /usr/local/hps/java/lcio-2.7.4-SNAPSHOT-bin.jar
 detector_dir = /usr/local/hps/java/detector-data/detectors
 

